[fortinet_fortiproxy]: update documentation

pkoutsovasilis opened the pull request:
<!-- Type of change
Please label this PR with one of the following labels, depending on the scope of your change:
- Bug
- Enhancement
- Breaking change
- Deprecation
-->

## Proposed commit message

update Documentation to remove sensitive information

<!-- Mandatory
Explain here the changes you made on the PR.

Please explain:

- WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
- WHY:  the rationale/motivation for the changes

This text will be pasted into the squash dialog when the change is committed and will be
a long term historical record of the change to help future contributors understand the
change, please help them by making it clear and comprehensive, they may be you.

If the commit title is adequate to describe both of these things, The text here may be omitted
or replaced with "See title". The title of the PR will be used as the commit message title when
the merge is made and the "See title" marker will be removed if present.

The text here and the PR title will be subject to the PR review process.
-->

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

<!-- Recommended
Add a checklist of things that are required to be reviewed in order to have the PR approved
-->
N/A

## How to test this PR locally

<!-- Recommended
Explain here how this PR will be tested by the reviewer: commands, dependencies, steps, etc.
-->

`elastic-package test`

## Related issues

<!-- Recommended
Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Closes #123
- Relates #123
- Requires #123
- Supersedes #123
-->
- Relates https://github.com/elastic/integration-docs/issues/429

## Screenshots

<!-- Optional
Add here screenshots presenting:
- Kibana UI forms presenting configuration options exposed by the integration
- dashboards with collected metrics or logs
-->
N/A

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/12343)
* Commit: d7993cb6af3eb73eddf49e7bffc3aac9beafd8dc

### Failed CI Steps
* [Check integrations fortinet_fortiproxy](https://buildkite.com/elastic/integrations/builds/12343#018ff23f-ca27-4c2c-833c-9b22ff53f4ef)

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12343","number":12343,"commit":"d7993cb6af3eb73eddf49e7bffc3aac9beafd8dc"}],"number":12343}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/12345)
* Commit: 1c94f949b4460fbaba735da217b07536d0dea750

### History
* :broken_heart: [Build #12343](https://buildkite.com/elastic/integrations/builds/12343) failed d7993cb6af3eb73eddf49e7bffc3aac9beafd8dc

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12345","number":12345,"commit":"1c94f949b4460fbaba735da217b07536d0dea750"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12343","number":12343,"commit":"d7993cb6af3eb73eddf49e7bffc3aac9beafd8dc"}],"number":12345}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/sec-deployment-and-devices (Team:Security-Deployment and Devices)

pkoutsovasilis commented:
> I changed my mind, let's remove these for now, I'll follow up with generic versions of these logs. (recreated from scratch)

Ok removed the respective entries and replaced them with samples from others in docs/README.md and sample_event.json

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/12356)
* Commit: 15c1206468587e16d530cc095a7e999bca7546d3

### Failed CI Steps
* [Check integrations fortinet_fortiproxy](https://buildkite.com/elastic/integrations/builds/12356#018ff303-4043-4a1f-bc6a-cd82cd826c0b)

### History
* :green_heart: [Build #12345](https://buildkite.com/elastic/integrations/builds/12345) succeeded 1c94f949b4460fbaba735da217b07536d0dea750
* :broken_heart: [Build #12343](https://buildkite.com/elastic/integrations/builds/12343) failed d7993cb6af3eb73eddf49e7bffc3aac9beafd8dc

cc @pkoutsovasilis

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12356","number":12356,"commit":"15c1206468587e16d530cc095a7e999bca7546d3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12345","number":12345,"commit":"1c94f949b4460fbaba735da217b07536d0dea750"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12343","number":12343,"commit":"d7993cb6af3eb73eddf49e7bffc3aac9beafd8dc"}],"number":12356}
buildkite-pr-comment-->

pkoutsovasilis commented:
@taylor-swanson your [commit](https://github.com/elastic/integrations/pull/10098/commits/2db456ed9b5f1b9036914a975ceb5b7291659442) LGTM! unfortunately for obvious reasons I can't approve it 😆

elasticmachine commented:
### :rocket: Benchmarks report




To see the full report comment with `/test benchmark fullreport`
<!--COMMENT_GENERATED_WITH_ID_benchmark-report-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/12358)
* Commit: 2db456ed9b5f1b9036914a975ceb5b7291659442

### History
* :broken_heart: [Build #12356](https://buildkite.com/elastic/integrations/builds/12356) failed 15c1206468587e16d530cc095a7e999bca7546d3
* :green_heart: [Build #12345](https://buildkite.com/elastic/integrations/builds/12345) succeeded 1c94f949b4460fbaba735da217b07536d0dea750
* :broken_heart: [Build #12343](https://buildkite.com/elastic/integrations/builds/12343) failed d7993cb6af3eb73eddf49e7bffc3aac9beafd8dc

cc @pkoutsovasilis

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12358","number":12358,"commit":"2db456ed9b5f1b9036914a975ceb5b7291659442"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12356","number":12356,"commit":"15c1206468587e16d530cc095a7e999bca7546d3"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12345","number":12345,"commit":"1c94f949b4460fbaba735da217b07536d0dea750"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/12343","number":12343,"commit":"d7993cb6af3eb73eddf49e7bffc3aac9beafd8dc"}],"number":12358}
buildkite-pr-comment-->

elasticmachine commented:
Package fortinet_fortiproxy - 0.1.1 containing this change is available at https://epr.elastic.co/search?package=fortinet_fortiproxy
